groovyw.bat

```diff
@@ -58,7 +58,24 @@ set _SKIP=2
 :win9xME_args_slurp
 if "x%~1" == "x" goto execute
 
-set CMD_LINE_ARGS=%*
+@REM set CMD_LINE_ARGS=%*
```

Comment:
Could you add a comment above this block to help spell out what it does? In particular since this is probably the first major custom bit of scripting added to the `groovyw` scripts themselves - I've otherwise tried to keep them as close to their `gradlew` parents as possible :-)